simplivity/resources/resource.py

```diff
@@ -212,15 +212,19 @@ def task_affected_resources(self, task, timeout):
 
         return affected_resources
 
-    def do_get(self, uri):
+    def do_get(self, uri, filters=None):
```

Comment:
I think these kinds of changes should go as separate PRs

Reply:
Hi @sijeesh ,
Since this enhancement was discovered during the implementation of current API, however the fix has dependency on overall working of this PR code. 
Even we thought of creating two different PRs however with this dependency the PR code can be merged and will not be completely functional hence we merged the code in same PR using two distinct commits. 

Please let us know your views on it, also if needed we can still go ahead and create two different PRs. 

Thank you looking forward to your views on it.